build-system/tasks/sweep-experiments/index.js

```diff
@@ -0,0 +1,445 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const argv = require('minimist')(process.argv.slice(2));
+const fastGlob = require('fast-glob');
+const log = require('fancy-log');
+const path = require('path');
+const {getOutput} = require('../../common/process');
+const {magenta, cyan} = require('ansi-colors');
+const {readJsonSync, writeFileSync} = require('fs-extra');
+
+const containRuntimeSource = ['3p', 'ads', 'extensions', 'src', 'test'];
+const containExampleHtml = ['examples', 'test'];
+
+const experimentsConfigPath = 'tools/experiments/experiments-config.js';
+const prodConfigPath = 'build-system/global-configs/prod-config.json';
+const canaryConfigPath = 'build-system/global-configs/canary-config.json';
+
+const globalWritablePaths = [
+  experimentsConfigPath,
+  prodConfigPath,
+  canaryConfigPath,
+];
+
+/**
+ * @param {string} cmd
+ * @return {?string}
+ */
+function getStdout(cmd) {
+  const {stdout, stderr} = getOutput(cmd);
+  if (!stdout && stderr) {
+    throw new Error(`${cmd}\n\n${stderr}`);
+  }
+  return stdout && stdout.trim();
+}
```

Comment:
Can we use the `getStdout` in `build-system/common/process.js` in place of this function? (The underlying `spawnSync` returns the status code that can be used for error checking.)

Reply:
Looks like `getStdout` returns `ChildProcess.stdout` and error code is part of the `ChildProcess` itself, so it's not received by caller. I also thought it was useful to print the command when failing. I'd be happy to rename the util, though.

Reply:
Makes sense. Yes to renaming if it isn't too much trouble.